components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -114,6 +114,7 @@ - (instancetype)initWithTitle:(NSString *)title message:(NSString *)message {
     _header.title = [title copy];
     _header.message = [message copy];
     self.backgroundColor = [UIColor whiteColor];
```

Comment:
Please don't call `self.backgroundColor` in the initializer. I see it's a custom setter and the object isn't finished being initialized yet.